Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+The goal of this project is deliver a functioning mock rover. The rover will allow the team to develop skills and knowledge to build a more feature completer rover in the future. The details are defined in the minimum viable product section. 
+
+%\begin{figure}
+%  \includegraphics[width=\linewidth]{figs/project-life-cycle.png}
+%  \caption{A PDD is the first piece of documentation to be archived in the project life cycle. Since the life cycle can be iterative, a new design document may also refer to one or more previous SPPs.}
+%\label{fig:lifecycle}
+%\end{figure}
+
+\section{Secondary Objectives}
+\label{sec:secondary-obj}
+  % Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+The study will also investigate the University Rover Competition (URC) as hosted by the mars Society as a potential long term goal. The competition is held annually and features 4 very intense competitions. Such a rover would need to be fully self navigating, perform scientific analysis on soil samples, and having a robotic arm capable of fine motor control. Such a project would be among the most ambitious SPEX has ever attempted. The skills learned from this project should be applicable to such a future project. 
+
+\section{Minimum Viable Product}
+\label{sec:mvp}
+  % Mvp
+  The minimum viable product (MVP) is defined by the following features: 
+
+\begin{itemize}
+  \item Rocker-bogie-like suspension
+  \item Ability to control remotely (such as a gamepad)
+  \item Ability to traverse terrain such as gravel, cement, and asphalt
+  \item Technical report and rover git as deliverable
+  \item Physical rover demo present at Imagine RIT
+\end{itemize}
+
+  The MVP is intended to be flexible in implementation depending on funding. The size and materials are not defined because of this. Should the funding allow for it there are many improvements that can be made beyond the MVP and quality materials. 
+
+\begin{itemize}
+  \item Lidar based CV for autonomous or partially autonomous navigation
+  \item Improved navigation with use of GPS system
+  \item Robotic arm with 3+ degrees of freedom
+    \item Drill or grasper attachment
+  \item Soil sample analysis with archemetes screw
+  \item Larger / faster rover
+  \item Solar cells for additional long-term navigation
+  \item better control interface  
```

Comment:
How is a better control interface defined? Is that wireless from wired? Is it computer GUI or app control? Does that mean basic automation support or programmed routes versus direct control?